plutus-ledger-constraints/src/Ledger/Constraints/OffChain.hs

```diff
@@ -333,6 +351,7 @@ initialState = ConstraintProcessingState
     , cpsMintRedeemers = mempty
     , cpsValueSpentBalancesInputs = ValueSpentBalances mempty mempty
     , cpsValueSpentBalancesOutputs = ValueSpentBalances mempty mempty
+    , cpsParams = def
```

Comment:
Shouldn't we pass it directly?

Reply:
It's not used here. It is passed directly in plutus-tx-constraints, where it is actually used. I'll add a comment.